src/pages/groupComparison/pathwayMapper/GroupComparisonPathwayMapper.tsx

```diff
@@ -63,9 +63,11 @@ export default class GroupComparisonPathwayMapper extends React.Component<
     }
 
     @computed get alterationEnrichmentRowData(): AlterationEnrichmentRow[] {
-        return this.props.store.alterationEnrichmentRowData.filter(gene => {
-            return this.activeGenes.includes(gene.hugoGeneSymbol);
-        });
+        return this.props.store.alterationEnrichmentRowData.result!.filter(
```

Comment:
let's assume that result might be undefined
```suggestion
        return (this.props.store.alterationEnrichmentRowData.result || []).filter(
```